pages/dkp/kommander/1.4/release-notes/1.4.4/index.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.4.4
+title: Release Notes Kommander 1.4.4
+menuWeight: 40
+excerpt: View release-specific information for Kommander 1.4.4
+enterprise: false
+---
+
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Kommander&reg; version 1.4.4 was released on February 17th, 2022.**
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/1.8/download/) and [install](/dkp/konvoy/1.8/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+### Supported versions
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**  | 1.18.x  |
+| **Maximum**  | 1.20.x  |
+| **Default**  | 1.20.5  |
+
+## Features/Improvements
+
+- Size of air-gapped bundles have been reduced by updating federated addons and images to those used in Kubernetes Base Addons (KBA)
+
+### Bug fixes
+
+- Replace has been added as a concurrency policy to to the Grafana dashboard CronJob to prevent the unintended creation of an unbound number of pods.
```

Comment:
```suggestion
- A concurrency policy has been added to to the Grafana dashboard CronJob to prevent the unintended creation of an unbound number of pods. (COPS-7105)
```

Reply:
Double "to".